frontend/frontend/src/components/Login.vue

```diff
@@ -70,5 +77,35 @@
 </script>
 
 <style>
+.Login {
+    display: flex;
+    flex-direction: column;
+}
+.input {
+	margin: 10px 20px 10px 20px;
+	padding-left: 10px;
+	height: 20px;
+	background-color: transparent;
+	border: 1px solid #1b1b1b;
```

Comment:
I'd recommend setting the width of the input equal to that of the buttons, when I looked at it on a larger screen the input appeared to take up the entire width of the screen.

Reply:
thanks for your feedback Fatimah. I implemented your style change requests. It should look a lot nicer now!

Reply:
Wonderful!